src/ScanItems.js

```diff
@@ -196,14 +197,25 @@ class ScanItems extends React.Component {
   };
 
   getRequestData(barcode) {
-    const { stripes, patron } = this.props;
+    const { stripes, patron, proxy } = this.props;
     const servicePointId = get(stripes, 'user.user.curServicePoint.id', '');
+    let data;
+    if (Object.keys(proxy).length > 0) {
```

Comment:
@K-Felk could we refactor this a bit with something similar to?

````js
const data = {
  itemBarcode: barcode.trim(),
  userBarcode: patron.barcode,
  servicePointId,
};

if (proxy.?barcode) {
  data.proxyUserBarcode = proxy.barcode;
}
````

Reply:
Normally this would be no problem, but my development environment blew up today, and I haven't been able to get it working again, so I can't run or test any changes...and I'll be out starting tomorrow for holidays/vacation.  I'll go ahead and merge this as-is, because I tested it and I know it works-I can leave a note to refactor when I get back (and figure out what's going on with my development platform).

Reply:
No worries @K-Felk :) Enjoy your time off!